peregrine/resources/submission/graphql/node.py

```diff
@@ -1065,7 +1061,14 @@ def instantiate_graphene(t):
 
 
 def resolve_datanode(self, info, **args):
-    """The root query for the :class:`DataNode` node interface.
+    """
+    The root query for the :class:`DataNode` node interface.
+
+    NOTE: A limitation of `datanode` is that the `first` and `offset` filters are not applied
```

Comment:
Thank you for adding this doc string. It highlights an otherwise non-obvious quirk to the querying behavior.